crates/aptos/src/move_tool/manifest.rs

```diff
@@ -74,6 +74,10 @@ pub struct Dependency {
     pub rev: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub subdir: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub aptos: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub address: Option<String>,
```

Comment:
We should probably document these especially since `aptos` and `address` are less straightforward?

Reply:
Absolutely! My plan was to dark launch for todays devnet, and then start with the documentation for next week.